examples/bicluster/plot_spectral_biclustering.py

```diff
@@ -3,63 +3,111 @@
 A demo of the Spectral Biclustering algorithm
 =============================================
 
-This example demonstrates how to generate a checkerboard dataset and
-bicluster it using the Spectral Biclustering algorithm.
-
-The data is generated with the ``make_checkerboard`` function, then
-shuffled and passed to the Spectral Biclustering algorithm. The rows
-and columns of the shuffled matrix are rearranged to show the
-biclusters found by the algorithm.
-
-The outer product of the row and column label vectors shows a
-representation of the checkerboard structure.
+This example demonstrates how to generate a checkerboard dataset and bicluster
+it using the Spectral Biclustering algorithm.
 
+The data is generated, then shuffled and passed to the Spectral Biclustering
+algorithm. The rows and columns of the shuffled matrix are then rearranged to
+plot the biclusters found by the algorithm.
 """
 
 # Author: Kemal Eren <kemal@kemaleren.com>
 # License: BSD 3 clause
 
-import numpy as np
-from matplotlib import pyplot as plt
+# %%
+# Generate sample data
+# --------------------
+# We generate the sample data using the `make_checkerboard` function. Each pixel
```

Comment:
It'd be nicer to make the hyperlinks to methods work here, for example:

```suggestion
# We generate the sample data using the :func:`~sklearn.datasets.make_checkerboard` function. Each pixel
```

(making sure the lines are not too long of course).

The `~` at the beginning of the above form makes sure the hyper link is rendered as `make_checkerboard` instead of the full path `sklearn.datasets.make_checkerboard` which is more verbose.